bart/sched/pelt.py

```diff
@@ -0,0 +1,347 @@
+#    Copyright 2015-2016 ARM Limited
+#    Copyright 2016 Google Inc. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""
+PELT Simulation Model
+"""
+
+from collections import namedtuple as _namedtuple
+from pandas import DataFrame as _DataFrame
```

Comment:
that also struck me as unnecessarily odd.  What's wrong with `pd.DataFrame`?  Furthermore, what's wrong with `namedtuple`?